lib/Styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+import { Elements } from './Attributes';
+
+export interface ComponentTheme {}
+
+export interface DivTheme {
+    colorTextRest: string;
+    colorBackground: string;
+}
+export interface ButtonTheme {
+    colorRest: string;
+    colorHover: string;
+    colorDisabled: string;
+    colorTextRest: string;
+    colorTextDisabled?: string;
+}
+
+const div = styled(Elements.div)`
+    &&&&&&& {
```

Comment:
what are the `&` for?

Reply:
this to bump up the CSS specificity in current element